cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NewTwoWindingsTransformerConversion.java

```diff
@@ -0,0 +1,426 @@
+                phaseTapChanger1 = combineTapChangers(cgmesT2xModel.end1.phaseTapChanger, cgmesT2xModel.end2.phaseTapChanger);
+            } else {
+                ratioTapChanger2 = combineTapChangers(cgmesT2xModel.end2.ratioTapChanger, cgmesT2xModel.end1.ratioTapChanger);
+                phaseTapChanger2 = combineTapChangers(cgmesT2xModel.end2.phaseTapChanger, cgmesT2xModel.end1.phaseTapChanger);
+            }
+        }
+
+        if (alternative.isXfmr2PhaseNegate()) {
+            negatePhaseTapChanger(phaseTapChanger1);
+            negatePhaseTapChanger(phaseTapChanger2);
+        }
+
+        AllTapChanger allTapChanger = new AllTapChanger();
+        allTapChanger.ratioTapChanger1 = ratioTapChanger1;
+        allTapChanger.phaseTapChanger1 = phaseTapChanger1;
+        allTapChanger.ratioTapChanger2 = ratioTapChanger2;
+        allTapChanger.phaseTapChanger2 = phaseTapChanger2;
+
+        return allTapChanger;
+    }
+
+    private AllShunt shuntAlternative(CgmesT2xModel cgmesT2xModel, Conversion.Config alternative) {
+        double g1 = 0.0;
+        double b1 = 0.0;
+        double g2 = 0.0;
+        double b2 = 0.0;
+        if (alternative.isXfmr2ShuntEnd1()) {
+            g1 = cgmesT2xModel.end1.g + cgmesT2xModel.end2.g;
+            b1 = cgmesT2xModel.end1.b + cgmesT2xModel.end2.b;
+        } else if (alternative.isXfmr2ShuntEnd2()) {
+            g2 = cgmesT2xModel.end1.g + cgmesT2xModel.end2.g;
+            b2 = cgmesT2xModel.end1.b + cgmesT2xModel.end2.b;
+        } else if (alternative.isXfmr2ShuntEnd1End2()) {
+            g1 = cgmesT2xModel.end1.g;
+            b1 = cgmesT2xModel.end1.b;
+            g2 = cgmesT2xModel.end2.g;
+            b2 = cgmesT2xModel.end2.b;
+        } else {
+            g1 = (cgmesT2xModel.end1.g + cgmesT2xModel.end2.g) * 0.5;
+            b1 = (cgmesT2xModel.end1.b + cgmesT2xModel.end2.b) * 0.5;
+            g2 = (cgmesT2xModel.end1.g + cgmesT2xModel.end2.g) * 0.5;
+            b2 = (cgmesT2xModel.end1.b + cgmesT2xModel.end2.b) * 0.5;
+        }
+
+        AllShunt allShunt = new AllShunt();
+        allShunt.g1 = g1;
+        allShunt.b1 = b1;
+        allShunt.g2 = g2;
+        allShunt.b2 = b2;
+
+        return allShunt;
+    }
+
+    private AllPhaseAngleClock phaseAngleClockAlternative(CgmesT2xModel cgmesT2xModel, Conversion.Config alternative) {
+        int phaseAngleClock1 = 0;
+        int phaseAngleClock2 = 0;
+
+        if (alternative.isXfmr2PhaseAngleClockOn()) {
+            phaseAngleClock1 = cgmesT2xModel.end1.phaseAngleClock;
+            phaseAngleClock2 = cgmesT2xModel.end2.phaseAngleClock;
+        }
+
+        AllPhaseAngleClock allPhaseAngleClock = new AllPhaseAngleClock();
+        allPhaseAngleClock.phaseAngleClock1 = phaseAngleClock1;
+        allPhaseAngleClock.phaseAngleClock2 = phaseAngleClock2;
+
+        return allPhaseAngleClock;
+    }
```

Comment:
Can be static.
Also, as said above, it may be more appropriate as post-processing

Reply:
Agree to manage `PhaseAngleClock` as post-processing. All conversion code associated with `PhaseAngleClock` has been deleted. A new PR will be done as post-processing